src/assets/_nav.scss

```diff
@@ -113,11 +120,34 @@
 .mobile {
   display: none;
   position: absolute;
-  right: 51px;
+  right: 10px;
   top: 0;
 }
-
-@media screen and (max-width: 745px) {
+@media screen and (max-width: 820px) {
+  .navBar {
+    .navLink {
+      margin: 0 20px;
+    }
+  }
+}
+@media screen and (max-width: 691px) {
```

Comment:
these seemingly random numbers make the OCD inside me sad, but i presume there's nothing that can be done about it... grr css?